tutteli-gradle-spek/src/main/groovy/ch/tutteli/gradle/spek/SpekPlugin.groovy

```diff
@@ -53,6 +53,7 @@ class SpekPlugin implements Plugin<Project> {
                     }
 
                     testImplementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+                    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$JUNIT_PLATFORM_VERSION"
```

Comment:
spek does not require the jupiter-engine

Reply:
Spek v1 does. It will fail with this exception if `junit-jupiter-engine` is not on the classpath:

```
Caused by: java.lang.ClassNotFoundException: org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine
```

Reply:
Oh... now I remember why I did not switch immediately to the build in runner when gradle 5 was out. First there was a jacoco issue (starting with Gradle 4.8) which I have fixed by now. But the other was, that I was not able to configure gradle in such a way that it spits out the exception in the console as you show above. How did you manage to do that? The configuration for `testLogging` you added to `SpekPluginIntTest` does not seem to work for me.
I'll investigate here a bit more once gradle shows me the exception :laughing:

Reply:
I do not know how to make this exception turn up on the command line. It seems to be particularly difficult because it is thrown in the _inner_ gradle instance, but already while setting up the tests.

What I did instead is looking at the generated HTML report inside the temporary gradle folder.

I actually prefer those HTML reports over e command line output because it is easier to navigate them and they are way less crowded. However, I do not think there is a good way to publish this report from Travis CI.

Reply:
I see now that it mislead me, I thought it would not even report anything in a normal gradle build but that seems possible, so fine with me